source/standalone/workflows/skrl/train.py

```diff
@@ -119,9 +120,14 @@ def main(env_cfg: ManagerBasedRLEnvCfg | DirectRLEnvCfg | DirectMARLEnvCfg, agen
     if args_cli.ml_framework.startswith("jax"):
         skrl.config.jax.backend = "jax" if args_cli.ml_framework == "jax" else "numpy"
 
+    # process seed from command line
+    if args_cli.seed == -1:
+        args_cli.seed = np.random.randint(0, 10000)
```

Comment:
would recommend using python standard library instead of numpy for this: 

https://docs.python.org/3/library/random.html